packages/manager/src/features/CloudPulse/Alerts/CreateAlert/CreateAlertDefinition.tsx

```diff
@@ -0,0 +1,262 @@
+import { Controller, FormProvider, useForm } from 'react-hook-form';
+import { useHistory } from 'react-router-dom';
+
+import { ActionsPanel } from 'src/components/ActionsPanel/ActionsPanel';
+import { Breadcrumb } from 'src/components/Breadcrumb/Breadcrumb';
+import { Drawer } from 'src/components/Drawer';
+import { Paper } from '@linode/ui';
+import { TextField } from 'src/components/TextField';
+import { Typography } from 'src/components/Typography';
+import {
+  useCreateAlertDefinition,
+  useNotificationChannels,
+} from 'src/queries/cloudpulse/alerts';
+import { useDatabaseEnginesQuery } from 'src/queries/databases/databases';
+
+import { MetricCriteriaField } from './Criteria/MetricCriteria';
+import { TriggerConditions } from './Criteria/TriggerConditions';
+import { CloudPulseAlertSeveritySelect } from './GeneralInformation/AlertSeveritySelect';
+import { EngineOption } from './GeneralInformation/EngineOption';
+import { CloudPulseRegionSelect } from './GeneralInformation/RegionSelect';
+import { CloudPulseMultiResourceSelect } from './GeneralInformation/ResourceMultiSelect';
+import { CloudPulseServiceSelect } from './GeneralInformation/ServiceTypeSelect';
+import { AddChannelListing } from './NotificationChannel/AddChannelListing';
+import { AddNotificationChannel } from './NotificationChannel/AddNotificationChannel';
+
+import type {
+  CreateAlertDefinitionPayload,
+  MetricCriteria,
+  NotificationChannel,
+  TriggerCondition,
+} from '@linode/api-v4/lib/cloudpulse/types';
+
+const triggerConditionInitialValues: TriggerCondition = {
+  evaluation_period_seconds: 0,
+  polling_interval_seconds: 0,
+  trigger_occurrences: 0,
+};
+const criteriaInitialValues: MetricCriteria[] = [
+  {
+    aggregation_type: '',
+    dimension_filters: [],
+    metric: '',
+    operator: '',
+    threshold: 0,
+  },
+];
+export const initialValues: CreateAlertDefinitionPayload = {
+  channel_ids: [],
+  label: '',
+  resource_ids: [],
+  rule_criteria: { rules: criteriaInitialValues },
+  service_type: '',
+  severity: '',
+  triggerCondition: triggerConditionInitialValues,
+};
+
+const generateCrumbOverrides = () => {
+  const overrides = [
+    {
+      label: 'Definitions',
+      linkTo: '/monitor/cloudpulse/alerts/definitions',
+      position: 1,
+    },
+    {
+      label: 'Details',
+      linkTo: `/monitor/cloudpulse/alerts/definitions/create`,
+      position: 2,
+    },
+  ];
+  return { newPathname: '/Definitions/Details', overrides };
+};
+
+export const CreateAlertDefinition = React.memo(() => {
+  const {
+    data: engineOptions,
+    isError: engineOptionError,
+    isLoading: engineOptionLoading,
+  } = useDatabaseEnginesQuery(true);
+
+  // const {
+  //   data: notificationChannels,
+  //   isError: notificationChannelError,
+  //   isLoading: notificationChannelLoading,
+  // } = useNotificationChannels();
+
+  const history = useHistory();
+  const alertCreateExit = () => {
+    const pathParts = location.pathname.split('/');
```

Comment:
No need to compute the url to go back, direct URL can be used ,as we are using at other places.